src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/service/NonAssociationsService.kt

```diff
@@ -66,7 +66,7 @@ class NonAssociationsService(
     offenderSearch.searchByPrisonerNumbers(prisonersToKeepApart)
 
     val nonAssociationJpa = createNonAssociationRequest.toNewEntity(
-      authorisedBy = authenticationFacade.currentUsername
+      updatedBy = authenticationFacade.currentUsername
```

Comment:
Could this also use `authenticationFacade.getUserOrSystemInContext()`? Or is this using `currentUsername` directly for some reason?

Reply:
No because it needs to throw an exception if not set